flash/text/seq2seq/core/data.py

```diff
@@ -62,290 +62,121 @@ def __setstate__(self, state):
 
 class Seq2SeqInput(Input):
     @requires("text")
-    def __init__(
+    def load_data(
```

Comment:
I would name this one, `BaseSeq2SeqInput` as I don't expect users to provide their HF dataset, but I might be wrong.

Reply:
Done